### DISCUSSION TITLE
How to backup settings

### DISCUSSION THREAD

**Discussion by FajnB** [koreader/koreader#8885] [Q&A] [answered]
Hi,

I was wondering, is there any way to completely export settings from one device to other, like gestures, preferences etc. everything I changed.

And most importantly, if yes, is it possible to import settings for example from pocketbook to android?

Thanks.

**Comment by Frenzie** [comment, marked as answer]
settings.reader.lua and the /settings directory will do that, but you might wish to only manually copy over gestures.
